### ISSUE TITLE
 How tags are allocated between 4 css categories?

### ISSUE THREAD

**Issue by Elijbet** [mbleigh/acts-as-taggable-on#827]
I am using your gem. Trying to understand how tags are allocated between 4 css categories. As in this example: https://github.com/mbleigh/acts-as-taggable-on#tag-cloud-calculations. 

What is the algorithm that decides the number of tags allocated to each css category? And can I change the number of css categories from 4 to 6 for example, can I do this: 

<% tag_cloud(@tags, %w(css1 css2 css3 css4 css5 css6)) do |tag, css_class| %> <%= link_to tag.name, { :action => :tag, :id => tag.name }, :class => css_class %> <% end %> 

Basically, how is the assignment of css categories distributed? I’ve looked through the documentation and have yet to find an answer.

**Comment by syguer** [comment]
The algorithm is [here](https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/tags_helper.rb#L10-L11).
